### PR TITLE
Add guild member flags

### PR DESCRIPTION
**Pull request by NoComment1105**
https://github.com/discord/discord-api-docs/pull/5874
I don't think this is everything, let me know if it isn't

### CODE REVIEW

**Comment by Lukellmann** [comment]
```
2023-02-16T12:09:07.8811675Z GuildTest > GuildMember serialization() FAILED
2023-02-16T12:09:07.8812964Z     kotlinx.serialization.MissingFieldException: Field 'flags' is required for type with serial name 'dev.kord.common.entity.DiscordGuildMember', but it was missing at path: $
2023-02-16T12:09:07.8814057Z         at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:90)
2023-02-16T12:09:07.8815009Z         at kotlinx.serialization.json.Json.decodeFromString(Json.kt:95)
2023-02-16T12:09:07.8815479Z         at json.GuildTest.GuildMember serialization(GuildTest.kt:80)
2023-02-16T12:09:07.8815735Z 
2023-02-16T12:09:07.8815846Z         Caused by:
2023-02-16T12:09:07.8816691Z         kotlinx.serialization.MissingFieldException: Field 'flags' is required for type with serial name 'dev.kord.common.entity.DiscordGuildMember', but it was missing
2023-02-16T12:09:07.8817493Z             at kotlinx.serialization.internal.PluginExceptionsKt.throwMissingFieldException(PluginExceptions.kt:20)
2023-02-16T12:09:07.8818160Z             at dev.kord.common.entity.DiscordGuildMember.<init>(Member.kt:10)
2023-02-16T12:09:07.8818687Z             at dev.kord.common.entity.DiscordGuildMember$$serializer.deserialize(Member.kt:10)
2023-02-16T12:09:07.8819214Z             at dev.kord.common.entity.DiscordGuildMember$$serializer.deserialize(Member.kt:10)
2023-02-16T12:09:07.8819919Z             at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:70)
2023-02-16T12:09:07.8820447Z             ... 2 more
```

**Comment by NoComment1105** [comment]
Fixed tests i think

File: common/src/test/resources/json/guild/guildmember.json [hunk 1]
```diff
@@ -3,5 +3,6 @@
   "roles": [],
   "joined_at": "2015-04-26T06:26:56.936000+00:00",
   "deaf": false,
-  "mute": false
+  "mute": false,
+  "flags": 0
```
Comment (HopeBaron) [inline common/src/test/resources/json/guild/guildmember.json:7]: I think it would be better to use non-zero values

**Review by HopeBaron — CHANGES_REQUESTED** [review]

File: common/src/test/resources/json/guild/guildmember.json (hunk 1 above)
Comment (NoComment1105) [reply common/src/test/resources/json/guild/guildmember.json:7]: zero values are used in the other tests, hence i used zero here

**Pull request merged**